packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -0,0 +1,81 @@
+import {NetworkName, networksChainConfig} from "@lodestar/config/networks";
+import {LightNode} from "../../../interfaces.js";
+import {CliCommandOptions} from "../../../utils/command.js";
+
+export type StartArgs = {
+  port: number;
+  network: string;
+  "execution-rpc": string;
+  mode: "rest" | "p2p";
+  "beacon-rpc"?: string[];
+  "beacon-bootnode"?: string[];
+  checkpoint?: string;
+};
+
+export type StartOptions = {
+  network: NetworkName;
+  executionRpcUrl: string;
+  port: number;
+  mode: LightNode;
+  beaconRpcUrls: string[];
+  beaconBootNodes: string[];
+  checkpoint?: string;
```

Comment:
would prefer wsCheckpoint to reflect it shouldn't be older than weak subjectivity period, also reflect the same in description